docs/cn/instructions/eventmesh-runtime-quickstart.zh-CN.md

```diff
@@ -75,21 +75,7 @@ sh start.sh
 ```
 > 注：如果操作系统为Windows, 可能需要将文件分隔符换成\
 
-**2.3.3 配置build.gradle文件**
-
-通过修改dependencies，compile project 项来指定项目启动后加载的插件
-
-修改`eventmesh-starter`模块下面的`build.gradle`文件
-
-加载**RocketMQ**插件配置：
-
-```java
-dependencies {
-    compile project(":eventmesh-runtime"), project(":eventmesh-connector-rocketmq")
-}
-```
-
-**2.3.4 启动运行**
+**2.3.3 启动运行**
```

Comment:
I think you may don't need to change this place, what do you think?

Reply:
I just follow this guide to start the event mesh yesterday. I just found that user don't need to change the build.gradle like what you said. And the start param should be set just before you run the main class. Set them first will be a little weird.And I can't compile this project in java 11 at first time. I think it should be mentioned in the  quick start guide.

Reply:
Yes, I mean you may don't need to remove the line 78 to 92, this may cause conflict with another pr :)

Reply:
oops, maybe I can resolve the conflict after that pr is merged. It would be a little challenge to avoid the conflict in single pr.

Reply:
hahah